.gitignore

```diff
@@ -21,3 +21,6 @@ config/database.yml
 # Ignore local CarrierWave uploads
 public/uploads/**/*
 public/spec/**/*
+
+# Ignore VCR cassettes
+/vcr_cassettes
```

Comment:
@solipet are you sure you want to ignore this directory? By ignoring the cassettes directory, we will end up recording the cassettes every time the tests are run somewhere new. We'll miss out on VCR's goals of having tests that hit the internet be fast, deterministic, and accurate.